### ISSUE TITLE
Uninstrument Text and RichText

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/beers-law-lab#332]
... the next time that beers-law-lab and concentration are published. This will be an API change, and will require migration rules.

**Comment by zepumph** [comment]
https://github.com/phetsims/joist/issues/934

**Comment by pixelzoom** [comment]
I'm going to do this eagerly, to see how adding migration rules goes.  @samreid @zepumph FYI.

**Comment by pixelzoom** [comment]
I discussed associated derived string Properties with @arouinfar.  If they are used to labels a control, keep them instrumented. If they are are readouts of values that available elsewhere, remove instrumentation.

I should also inspect uses of DerivedStringProperty -- it looks like some of them should be PatternStringProperty.

**Comment by pixelzoom** [comment]
Done in the above commits.  This took me 1.5 hours, required consulting with @arouinfar, and required 18 new migration rules (13 TandemFragmentDelete, 5 TandemFragmentRenamed). 

This should be reviewed before the sim is republished, so I'll keep this issue open until then.

**Comment by pixelzoom** [comment]
For the 1.8 release, I'll need to check whether any migration rules are needed for these changes (as the first comment in this issue suggests).

**Comment by pixelzoom** [comment]
I noticed that there are still a number of DerivedStringProperty instances that should be uninstrumented. So self assigning.

**Comment by pixelzoom** [comment]
@DianaTavares This needs to be reviewed for the 1.8 release #339.

**Comment by pixelzoom** [comment]
No, this is not related to A11y View or description -- it's PhET-iO.   This sim was over-instrumented, so we removed some of that over-instrumentation, specifically for Text and RichText.   

If @arouinfar is available, it would make more sense to have her review this.  @arouinfar are you available?  You can see what was uninstrumented by reviewing the migration rules that were added in https://github.com/phetsims/phet-io-sim-specific/commit/346f8a32d2823c08b1872bf21a4115e9cdd81742.

**Comment by arouinfar** [comment]
dropperNode.labelText

**Comment by arouinfar** [comment]
@pixelzoom I reviewed main and compared it to 1.7 and the migration rules you linked. The changes look good overall. I found a few more instances of text that we might want to consider uninstrumenting, but I'll leave it up to you.

* `dropperNode.labelText`
* `shakerNode.labelText`
* `rulerNode.unitsLabelText`
* `numberDisplay.valueText` (this should probably be done in https://github.com/phetsims/scenery-phet/issues/812)
